eng/pipelines/templates/steps/java-publishing.yml

```diff
@@ -73,4 +74,5 @@ steps:
           -RepositoryUsername azuresdk
```

Comment:
You may need to put quotes around ` -GroupIDFilter "${{ parameters.GroupID }}"` in order for PS to accept the empty string.

Reply:
Trying decorating with an attribute to allow nulls, but will try this afterwards if that doesn't work.

Reply:
OK I think we're good.